hack/runDlv.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+DLV=$HOME/go/bin/dlv
+[[ -x $DLV ]] || \
+  sudo dnf install -y golang &&
+  go install github.com/go-delve/delve/cmd/dlv@latest
+
+sudo systemctl kill microshift
+sudo systemctl disable --now microshift
+sudo systemctl kill microshift
```

Comment:
Why 2 kills?

Reply:
That's a good question let's kill one kill :-)